### PR TITLE
Create blog post scrum basics series

### PR DESCRIPTION
**Pull request by BirgitBader**
@maletzke @steffimeier please give your feedback.

### CODE REVIEW

File: _posts/2015-10-06-scrum-basics-1.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+In an ideal world it would go from creating concepts, over developing and testing the product, to rollout, marketing and sales.
+But most companies “only” have the development part in their Scrum teams.
+The members of those teams should strive to get “T-shaped”.
+That means they not only have one field of expertise, but also a broad basis so they can help out in other areas in the team.
+That doesn’t mean everyone should know everything, it only means everyone is willing to do more than just their own thing.
+
+# Advantages
+
+The big advantage of those teams in comparison to assigning single persons to specific parts of the product development is, that you always have all the abilities needed in one room.
+In waterfall you often have the problem that the person who did the backend development is already in the next project when the designers realise that they need some more data from the backend.
```
Comment (maletzke) [inline _posts/2015-10-06-scrum-basics-1.md]: "Waterfall" should probabyl be explained

File: _posts/2015-10-06-scrum-basics-1.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+That means this list contains all the tickets that need to be done for a product to be developed.
+The Sprint Backlog is also a list of estimated, prioritised tickets, but it only contains those tickets that will be solved within the next iteration.
+
+In the middle you can see the iteration, in Scrum called Sprint, which is a period of time (at ePages it's 2 weeks) during which the tickets in the Sprint Backlog have to be done and the scope of the Sprint Backlog should not change.
+The development status of those tickets is represented on a so called Scrum Board or Sprint Board.
+It provides an overview of all tickets of one team for one Sprint including the status of those tickets (e.g. Open, In Progress, Review, QA, Done) and which team member is currently working on which ticket.
+With this Scrum Board everyone knows at all times about the development status and if the team is struggling or ahead of time.
+It also shows if one person is working on too many tickets at the same time or if the team has a long queue in QA or Review.
+
+At ePages we only have one team that is using a paper board:
```
Comment (maletzke) [inline _posts/2015-10-06-scrum-basics-1.md]: Might be interesting why this one team is using a paper board

**Comment by BirgitBader** [comment]
@JessTinker please proof-read, thanks

File: _posts/2015-10-13-scrum-basics-1.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+Well, part of my job is to teach new colleagues exactly that.
+Now we'd like to share this knowledge with you in our **Scrum Basics Series**.
+This series will consist of 6 parts and starts with the basic question:
+What is Scrum?
+
+So let’s start with a short definition:
+
+>“Scrum is a team based framework
+to develop complex products
+or systems in an iterative way.“
```
Comment (JessTinker) [inline _posts/2015-10-13-scrum-basics-1.md]: interactive

File: _posts/2015-10-13-scrum-basics-1.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+It should have all the abilities needed to fulfil the tasks, e.g. in software development a team consists of backend developers and frontend developers as well as designers and testers.
+In an ideal world it would go from creating concepts, over developing and testing the product, to rollout, marketing and sales.
+But most companies “only” have the development part in their Scrum teams.
+The members of those teams should strive to get “T-shaped”.
+That means they not only have one field of expertise, but also a broad basis so they can help out in other areas in the team.
+That doesn’t mean everyone should know everything, it only means everyone is willing to do more than just their own thing.
+
+# Advantages
+
+The big advantage of those teams in comparison to assigning single persons to specific parts of the product development is, that you always have all the abilities needed in one room.
```
Comment (JessTinker) [inline _posts/2015-10-13-scrum-basics-1.md]: The big advantage of those teams in comparison to assigning a single person to specific parts of the product development is that you always have all the abilities needed in one room.

File: _posts/2015-10-13-scrum-basics-1.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+In [waterfall](https://en.wikipedia.org/wiki/Waterfall_model) you often have the problem that the person who did the backend development is already in the next project when the designers realise that they need some more data from the backend.
```
Comment (JessTinker) [inline _posts/2015-10-13-scrum-basics-1.md]: is already working on the next project when the designers realise

File: _posts/2015-10-13-scrum-basics-1.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+In Scrum that won’t happen.
+Another advantage is that you always have several eyes on one problem.
+Solutions can be found faster if you have more experience in the room and people who get stuck at some point always have someone to ask right next to them.
+And don’t forget the much faster feedback cycle with the testers when they are right inside the team!
```
Comment (JessTinker) [inline _posts/2015-10-13-scrum-basics-1.md]: And let's not forget the much faster feedback cycle with the testers when they are right inside the team!

File: _posts/2015-10-13-scrum-basics-1.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+No ticket is really done until it has been tested.
+
+# How is Scrum made for developing complex products or systems?
+
+Complex problems are usually better solved if more minds are involved.
+An idea can be good but it will only become really great if you open it up to your team partners and get as many information and experiences involved as possible.
```
Comment (JessTinker) [inline _posts/2015-10-13-scrum-basics-1.md]: team partners and get as much information and experience involved as possible.

**Pull request merged**